src/simulation/simulation_store_common.jl

```diff
@@ -0,0 +1,180 @@
+# Keep these in sync with the Symbols in src/core/definitions.
+get_store_container_type(::AuxVarKey) = STORE_CONTAINER_AUX_VARIABLES
+get_store_container_type(::ConstraintKey) = STORE_CONTAINER_DUALS
+get_store_container_type(::ExpressionKey) = STORE_CONTAINER_EXPRESSIONS
+get_store_container_type(::ParameterKey) = STORE_CONTAINER_PARAMETERS
+get_store_container_type(::VariableKey) = STORE_CONTAINER_VARIABLES
+
+function write_model_dual_results!(
```

Comment:
I moved this code here because it is common to both subtypes of SimulationStore.

Reply:
ok.